pkgs/by-name/cl/clevis/package.nix

```
@@ -0,0 +1,125 @@
+{
+  lib,
+  asciidoc,
```

A way to get rid of this `callPackage`, and keep the "API" of this `asciidoc` argument is this:

```suggestion
  asciidoc-full,
  asciidoc ? asciidoc-full,
```

---

I'm concerned it can not work as intended. I will try to seek the PR I tried this and it didn't work.